src/utils/route-guard/AuthGuard.tsx

```diff
@@ -9,15 +9,33 @@ import { getFusionConfig } from "./FusionConfiguration"
 const AuthGuard = ({ children }: GuardProps) => {
   const { isAuthenticated } = useFusionAuth()
   const { enabled: fusionIsEnabled } = getFusionConfig()
+  const [isLoading, setIsLoading] = useState(true)
   const navigate = useNavigate()
   const location = useLocation()
 
   useEffect(() => {
-    if (fusionIsEnabled && fusionIsEnabled === "true" && !isAuthenticated) {
-      navigate("authentication/login")
+    const timer = setTimeout(() => {
```

Comment:
@astowny you must remove this horrible delay added by Benjamin.
It's not the problem because the loading time does not exist on the built version, which has only one Javascript file.

Reply:
What is causing this behaviour ?